src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = 40 * days;
+
+  if (days >= 7) {
+    result -= 50;
```

Comment:
Prefer 'if' with 'return' over 'if else' to simplify later conditions. You can return the result immediately after applying the discount for 7 or more days.